testcases/kdump/lib/crasher/crasher.c

```diff
@@ -37,22 +37,21 @@ void crasher_exit(void);
 module_init(crasher_init);
 module_exit(crasher_exit);
 
+
 #define CRASH "crasher"		/* name of /proc entry file */
 
-static int crasher_read(char *buf, char **start, off_t offset, int len,
-			int *eof, void *data)
+static int crasher_read(struct file *filp,char *buf,size_t count,loff_t *offp)
 {
 	return (sprintf(buf, "\n"));
 }
 
```

Comment:
So it just prints newline when the file is read? Why do we implement the read callback at all?